### ISSUE TITLE
Add readme with setup instructions

### ISSUE THREAD

**Issue by chrisdwheatley** [robwormald/ng2chat#2]
Would love to feature this in a list of Angular2 repo's I've put together but I'm not entirely sure how to get the project up and running. I've got as far as `jspm install` but I think `gulp` and the modules it's using need to be defined somewhere to be installed so I can run commands from the gulpfile.

**Comment by maku** [comment]
yeah files which are required in gulp should be added to package.json

**Comment by chrisdwheatley** [comment]
Yup, that's what I was alluding to, albeit in a rather longwinded fashion.

I may have some time to put together a PR for a basic `package.json` later today but it'd still be nice to have some setup instructions.
